openmdao/docs/basic_guide/basic_case_recording.rst

```diff
@@ -0,0 +1,41 @@
+.. _basic_case_recording:
+
+************************
+Basic Recording Example
+************************
+
+Recording Terminology
+---------------------
+
+| **Case**: A case is an OpenMDAO model.
+| **Case Recorder**: An OpenMDAO module used to store data about the optimization in an SQL file.
+| **Sources**: The different parts of an OpenMDAO model to probe for data. `Problem`, `Driver`, and `System`.
+
+Basic Recording Example
+------------------------
+
+Below is an basic example of how to create a recorder, attach it to a Problem, save the information,
+and retrieve the data from the recorder. `list_outputs` is a quick way to show all of your outputs
+and their final values, and should you need to isolate a single value OpenMDAO provides two ways to
```

Comment:
Not their "final" values.. just their values at the time the Case was recorded...